docs/api/howtos/create-container.mdx

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_label: Starting a Docker container
+title: Starting a Docker container using the API
+description: Learn how to start a custom Docker container via the mStudio API
+tags:
+  - Docker
+  - Containers
+---
+
+import OperationExample from "@site/src/components/OperationExample";
+import OperationLink from "@site/src/components/OperationLink";
+
+## Prerequisites
+
+To start a custom Docker container, you will need to have the following:
+
+- The project ID of an existing project ([how to create a project](../create-project)), in which the container hosting feature is available
+- A container image that you want to start. This image may be available in any public or private registry; however, the registry must be reachable from the internet and not hidden behind a firewall.
+
+## Defining a private registry (optional)
+
+If your container image should be loaded from a private registry, you first need to define this registry for the respective project.
```

Comment:
Wir haben bei einigen Usern das Phänomen gehabt, dass sie sich viel Arbeit damit gemacht haben, GitHub als Registry anlegen zu wollen – die Registry war aber schon angelegt. Sie hätten nur ihre Zugangsdaten hinterlegen müssen. Sollten wir hier darauf hinweisen, dass wir die gängisten Registries schon angelegt haben?

Reply:
Fixed in https://github.com/mittwald/developer-portal/pull/549/commits/bdd196ac7eef54d7caf8a1d447706c0b80e53982